project/src/store/root-reducer.ts

```diff
@@ -0,0 +1,18 @@
+import {combineReducers} from 'redux';
+import {userProcess} from './user-process/user-process';
+import { offerProperty } from './offer-property/offer-property';
+import { offerData } from './offer-data/offer-data';
+
+export enum NameSpace {
+  data = 'DATA',
```

Comment:
Ключи енама  с большой буквы
```suggestion
  Data = 'DATA',
```